Test-№2.0-from-23.11.2022/task1.c

```diff
@@ -0,0 +1,250 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#define dateSize 10

```

Comment:
```suggestion
#define DATE_SIZE 10
```
Так принято именовать define константы